tests/fast-integration/skr-test/test-own-cluster.js

```diff
@@ -0,0 +1,69 @@
+const {gatherOptions} = require('./index');
+const {getOrProvisionSKR} = require('./provision/provision-skr');
+const {GardenerClient, GardenerConfig} = require('../gardener');
+
+const {getSecret} = require('../utils');
+
+const {expect} = require('chai');
+
+const {toBase64} = require('../utils');
+
+const {genRandom} = require('../utils');
+
+const skipProvisioning = process.env.SKIP_PROVISIONING === 'true';
+const provisioningTimeout = 1000 * 60 * 30; // 30m
+const deprovisioningTimeout = 1000 * 60 * 95; // 95m
```

Comment:
wow, it really takes that long? shouldn't deprovisioning be faster? I'm inclined to say there must be a bug in the deprovisioning workflow if it takes 95 minutes, not sure whether it can be fixed here in the scope of the test.

Reply:
You are right, this is just a leftover from `test.js` that I based my work on. One minute should be sufficient. I am just marking shoot for removal not waiting for the result in tests.